README.md

```diff
@@ -80,9 +80,9 @@ The [.NET Framework Docker samples](https://github.com/Microsoft/dotnet-framewor
 
 ### Building .NET Framework Apps with Docker
 
-* [.NET Framework Console Docker Sample](dotnetapp/README.md) - This [sample](dotnetapp/Dockerfile) builds, tests, and runs the sample. It includes and builds multiple projects.
-* [ASP.NET Web Forms Docker Sample](aspnetapp/README.md) - This [sample](aspnetapp/Dockerfile) demonstrates using Docker with an ASP.NET Web Forms app.
-* [ASP.NET MVC Docker Sample](aspnetmvcapp/README.md) - This [sample](aspnetmvcapp/Dockerfile) demonstrates using Docker with an ASP.NET MVC app.
+* [.NET Framework Console Docker Sample](samples/dotnetapp/README.md) - This [sample](samples/dotnetapp/Dockerfile) builds, tests, and runs the sample. It includes and builds multiple projects.
```

Comment:
These should be fully qualified URLs (e.g. prefixed with `https://github.com/Microsoft/dotnet-framework-docker/tree/master/`.  The reason is that this readme gets copied to Docker Hub and used as the `Full Description` - https://hub.docker.com/r/microsoft/dotnet-framework/